src/adr/0040-generalize-event-bus.md

```diff
@@ -0,0 +1,27 @@
+#40. Generalize EventBus
+
+Date: 2020-06-11
+
+## Status
+
+Proposed
+
+## Context
+
+When searching for users by quota ratio if we set the value of the parameters to 0, the search feature is supposed to return newly created users
+who have not received any email yet at that point. However, this is not the case due to the quotas are currently being initialized only after
+an user receive the first email.
+
+In order to solve this problem, we need to initialize user quotas upon creation time. Event driven system allows indirection so that the users api 
```

Comment:
```suggestion
In order to solve this problem, we need to initialize user quotas upon user creation time. Event driven system allows indirection so that the users api 
```